Main.java

```diff
@@ -0,0 +1,69 @@
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class Main {
+    static HashMap<String, String> FirstHashMap = new HashMap<String, String>();
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        //Task 1.
+
+        FirstHashMap.put("978-1-60309-452-8", "A letter to Jo");
+        FirstHashMap.put("978-1-60309-459-7", "Lupus");
+        FirstHashMap.put("978-1-60309-444-3", "Red Panda and Moon Bear");
+        FirstHashMap.put("978-1-60309-461-0", "The Lab");
+        int programLoopValue = 0;
+        while(programLoopValue == 0){
```

Comment:
this value never changes, ide and checkstyle will remind you too, why u use it?

Reply:
287d4e7a